docs/html/classcachemere_1_1policy_1_1detail_1_1BloomFilter.html

```diff
@@ -0,0 +1,270 @@
+<tr class="memitem:a8e6097bc494369c9a6baff51b9777d76"><td class="memItemLeft" align="right" valign="top">bool&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#a8e6097bc494369c9a6baff51b9777d76">maybe_contains</a> (const Key &amp;key) const</td></tr>
+<tr class="memdesc:a8e6097bc494369c9a6baff51b9777d76"><td class="mdescLeft">&#160;</td><td class="mdescRight">Test membership of the specified key.  <a href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#a8e6097bc494369c9a6baff51b9777d76">More...</a><br /></td></tr>
+<tr class="separator:a8e6097bc494369c9a6baff51b9777d76"><td class="memSeparator" colspan="2">&#160;</td></tr>
+<tr class="memitem:a2f6d118fde3178d936610dd5f7266e1a"><td class="memItemLeft" align="right" valign="top">size_t&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#a2f6d118fde3178d936610dd5f7266e1a">memory_used</a> () const noexcept</td></tr>
+<tr class="memdesc:a2f6d118fde3178d936610dd5f7266e1a"><td class="mdescLeft">&#160;</td><td class="mdescRight">Get an estimate of the memory consumption of the filter.  <a href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#a2f6d118fde3178d936610dd5f7266e1a">More...</a><br /></td></tr>
+<tr class="separator:a2f6d118fde3178d936610dd5f7266e1a"><td class="memSeparator" colspan="2">&#160;</td></tr>
+<tr class="memitem:aac87cf9d8f32a4b8894b66accb6059c0"><td class="memItemLeft" align="right" valign="top">double&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#aac87cf9d8f32a4b8894b66accb6059c0">saturation</a> () const noexcept</td></tr>
+<tr class="memdesc:aac87cf9d8f32a4b8894b66accb6059c0"><td class="mdescLeft">&#160;</td><td class="mdescRight">Get the saturation of the filter.  <a href="classcachemere_1_1policy_1_1detail_1_1BloomFilter.html#aac87cf9d8f32a4b8894b66accb6059c0">More...</a><br /></td></tr>
+<tr class="separator:aac87cf9d8f32a4b8894b66accb6059c0"><td class="memSeparator" colspan="2">&#160;</td></tr>
+</table>
+<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
+<div class="textblock"><h3>template&lt;typename Key, typename KeyHash = std::hash&lt;Key&gt;&gt;<br />
+class cachemere::policy::detail::BloomFilter&lt; Key, KeyHash &gt;</h3>
+
+<p>Probabilistic datastructure for representing sets in a space-efficient format. </p>
+<p>A bloom filter is a constant-sized datastructure, which means that insertions will never make the filter allocate more memory. However, too many inserts will severly impact the accuracy of filter membership tests. </p><dl class="tparams"><dt>Template Parameters</dt><dd>
+  <table class="tparams">
+    <tr><td class="paramname">Key</td><td>The type of the items that will be inserted into the set. </td></tr>
```

Comment:
Great point. 
Actually, the name `Key` isn't super appropriate for this data structure - it makes a bit of sense because TinyLFU uses it to store keys, but it technically could be used to store any hashable type. I'll change the naming all around to `item`, and change the template parameter to `Item` (`Item` instead of `ItemType` because since it's a `typename` template parameter, we already know it's a type)